lib/paths.js

```diff
@@ -1,6 +1,6 @@
 const path = require('path');
 
 module.exports = {
-  PROJECT_ROOT: process.env.PROJECT_ROOT,
+  PROJECT_ROOT: process.env.PROJECT_ROOT ? process.env.PROJECT_ROOT : process.cwd(),
```

Comment:
Had this sitting in my checkout as a small fix @abutterworth and I talked about - threw it in to save tedium.

Reply:
process.env.PROJECT_ROOT can be undefined if you use any of this code outside of calling it through `fedx-scripts` - this is common if you're trying to extend a webpack config, for instance.